### DISCUSSION TITLE
[protocols/identify] Help: where to find the example of how to connect to a known PeerID(p2p multiaddr) using identify

### DISCUSSION THREAD

**Discussion by fishseabowl** [libp2p/rust-libp2p#2561] [General]
I'd appreciate any examples of connecting with a known PeerID to send and receive messages. thanks

**Comment by mxinden** [comment]
Does https://github.com/libp2p/rust-libp2p/blob/master/examples/ipfs-private.rs help? It combines `libp2p-identify`, `libp2p-gossipsub` and `libp2p-ping`.
